benchmark/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'java'
+apply plugin: 'application'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.google.caliper:caliper:1.0-beta-2'
+    compile 'com.google.guava:guava:18.0'
+
+    compile('com.google.api.grpc:grpc-google-cloud-pubsub-v1:0.1.5') {
```

Comment:
Is the concern that this will add a dependency to GAX? This only affects the benchmark subproject, so GAX itself will compile without this dependency.